### DISCUSSION TITLE
bios in .rom format not flashable on a gigabyte b550 motherboard

### DISCUSSION THREAD

**Discussion by keumarr** [terminatorul/NvStrapsReBar#63] [Q&A] [answered]
Hello, I am looking to activate the rebar on my gigabyte b550 gaming x v1 for my rtx 2060.

I managed to add NvStrapsReBar.ffs to the latest gigabyte F16 bios unfortunately UEFITool puts a .rom format in my bios which is basic in .f16 which means that the modified bios cannot be used when flashing.

How can I change the format of my modified bios file?

I have read a lot of things for asus msi but not gigabyte.

Thanks in advance.
![image](https://github.com/terminatorul/NvStrapsReBar/assets/163852659/1467be48-f932-4d3d-82ef-ef85096e820c)
![image1](https://github.com/terminatorul/NvStrapsReBar/assets/163852659/62f339a9-3649-4ec6-8eb2-ae7370e032c9)

**Comment by keumarr** [comment]
Would changing the format to .f16 be enough?

Edit: change the format .rom to .F16 for me don't work.

**Comment by TessellatedGuy** [comment]
I just renamed the .rom file to whatever the original BIOS file's name was, including the file extension. It worked fine on my MSI motherboard using M-Flash.

**Reply by keumarr** [reply]
I tried the 2 methods from bios and @bios the gigabyte app it doesn't work bios.rom or bios.f16

**Reply by TessellatedGuy** [reply]
Some people had success using "Q-Flash Plus" on gigabyte motherboards. Someone with a gigabyte B550 motherboard like yours also flashed the BIOS successfully using the same method: https://github.com/terminatorul/NvStrapsReBar/issues/1#issuecomment-2002895376

**Reply by keumarr** [reply]
Oh great, however it doesn't specify whether I should do it with my bios.rom or bios.f16 file. I will try both

**Reply by keumarr** [reply]
Ok neither of them is in .bin

**Reply by keumarr** [reply, marked as answer]
"resolved"
It's good I succeeded with q-flash plus just rename the bios file to gigabyte.bin, turn off your computer (I didn't need to remove cpu ram gpu etc) press the q flash plus button and leave for about 5-10min
On the other hand, your bios is reset carefully to save if you have overclocking profiles or other.

Thank you @TessellatedGuy for helping me, it's very kind.
![image](https://github.com/terminatorul/NvStrapsReBar/assets/163852659/3cce17e6-f9f3-492e-b60b-76566c94004c)
